src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure the href attribute points to the correct CSS file.